initramfs-init.in

```diff
@@ -618,6 +618,7 @@ mount -o bind /sys $sysroot/sys
 mount -o bind /proc $sysroot/proc
 mount -o bind /dev $sysroot/dev
 if [ -n "$ovlfiles" ]; then
+	apkflags="$apkflags --no-commit-hooks"
```

Comment:
I would like to instead change the `--force` flag to `--initramfs-diskless-boot` in edge. This commit might be suitable for stable trees, but perhaps then add the flag directly to the next line, or set it uncoditionally few lines earlier.

Reply:
@fabled agreed that's better, fixed.